src/Maker/MakeConvertServices.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Util\PhpServicesCreator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * @author Antoine Michelet <jean.marcel.michelet@gmail.com>
+ */
+final class MakeConvertServices extends AbstractMaker
+{
+    private $fileManager;
+    private $fileSystem;
+
+    public function __construct(Filesystem $fileSystem)
+    {
+        $this->fileSystem = $fileSystem;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:convert-services';
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command->setDescription('Converts services.yaml file in php');
+
+        $command->addArgument('save-service-yaml',
+            InputArgument::REQUIRED,
+            'Do you want to preserve services.yaml Y/n ?'
+        );
+
+        $command->addArgument('warning-comments',
+            InputArgument::REQUIRED,
+            'Warning: This command does not allow the conversion of custom comments in services.yaml. Do you want continue Y/n ?'
+        );
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        if ('n' === strtolower($input->getArgument('warning-comments'))) {
+            return;
+        }
+
+        $creator = new PhpServicesCreator();
+        $phpServices = $creator->convert(file_get_contents('config/services.yaml'));
```

Comment:
You should use `$this->fileManager->fileExists($path)` to check if the file exists and `$this->configUpdater->updateForUserClass(`.

If you're doing something "normal" (like reading a file from a user's project), it has *all* been done before - there is an example of how to do it somewhere in the bundle ;)

Reply:
??? I'm not sure what you want. i added a remove method to fileManager In the meantime...
`$this->configUpdater->updateForUserClass(`